### PR TITLE
Личный проект: максимум подвижности

### PR DESCRIPTION
**Pull request by akzhar**
---
:mortar_board: [Личный проект: максимум подвижности](https://up.htmlacademy.ru/javascript/17/user/490939/tasks/10)

:boom: https://htmlacademy-javascript.github.io/490939-keksobooking-17/5/

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -9,11 +9,16 @@ var MIN_PRICES = {
   house: 5000,
   bungalo: 0
 };
+var MAP_LIMITS = {
+  xMin: 0,
+  xMax: 1200,
+  yMin: 130,
+  yMax: 630,
+};
```
Comment (rkhaslarov) [inline js/main.js]: this can be the last solution, because map width is responsive, you should take care of it )

File: js/main.js [hunk 2]
```diff
@@ -126,7 +121,58 @@ function onTimeInOutSelectChange(evt) {
...
+    }
+    if (pinMainTop > (MAP_LIMITS.yMax - pinHeight)) {
+      pinMainTop = MAP_LIMITS.yMax - pinHeight;
+    }
+    if (pinMainLeft < (MAP_LIMITS.xMin - pinWidth / 2)) {
+      pinMainLeft = MAP_LIMITS.xMin - pinWidth / 2;
+    }
+    if (pinMainLeft > (MAP_LIMITS.xMax - pinWidth / 2)) {
+      pinMainLeft = MAP_LIMITS.xMax - pinWidth / 2;
+    }
```
Comment (rkhaslarov) [inline js/main.js]: moreover, I found the issue, after activating the map, I can drag the pin at all

Comment (rkhaslarov) [reply js/main.js]: Probably, because you removed the listener of mousedown event =)

Comment (akzhar) [reply js/main.js]: If I remove the listener you can NOT move the pin (that what I've done).
But specification says that we can move the pin more than once, I realize it now =)
I guess I must read the specification more carefully =)

File: js/main.js (hunk 1 above)
Comment (akzhar) [reply js/main.js]: Sure. Fixed.

**Pull request merged**